build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec

```diff
@@ -3,7 +3,7 @@
     <metadata minClientVersion="2.8.6">
         <title>Active Directory Authentication Library</title>
         <id>Microsoft.IdentityModel.Clients.ActiveDirectory</id>
-        <version>REPLACE_DURING_BUILD</version>
+        <version>0.0.1-LOCALBUILD</version>
```

Comment:
Changed the _version_ to a valid value so the project can be built locally without setting the version explicitly.